Vidstreaming/dist/metadata.json

```diff
@@ -3,16 +3,36 @@
     "name": "GURRMAN375D",
     "id": "446528923535278082"
   },
-  "url": "vidstreaming.io",
+  "service": "Vidstreaming",
   "description": {
     "en": "Vidstreaming is a website where you can find the latest anime released."
   },
-  "service": "Vidstreaming",
-  "version": "1.0.1",
+  "url": "vidstreaming.io",
+  "version": "2.1.0",
```

Comment:
I think you should just make this 2.0.0.

Reply:
Done